packages/amplify_api/example/pubspec.yaml

```diff
@@ -19,6 +19,8 @@ dependencies:
     # The example app is bundled with the plugin so we use a path dependency on
     # the parent directory to use the current plugin's version.
     path: ../
+  amplify_auth_cognito:
```

Comment:
Same question as above

Reply:
After discussing with @haverchuck we will be removing the Auth addition changes to the API example app as it's out of scope for the NS upgrade.